scripts/create-plugin.js

```diff
@@ -14,7 +14,7 @@ fs.mkdirSync(directory)
 process.chdir(directory)
 
 console.log('📦 initialising package')
-execSync('npm init -y --scope @dotcom-tool-kit')
+execSync('npm init -y --scope @dotcom-tool-kit@init')
```

Comment:
**chore**: this is unrelated (and will probably break!), this call is just for creating the `package.json` in a new plugin

**thought**: do we still use this script? does it even still work? maybe let's just delete it?

Reply:
oops yeah, thrown off by the scope argument! I feel like we only don't use this script because we aren't in the habit of creating new plugins 😅 but I think we should keep it, if not just to have a bit of documentation on all the things you need to look out for when creating a plugin.